ckeditor_filebrowser_filer/static/ckeditor/ckeditor/plugins/filerimage/dialogs/filerImageDialog.js

```diff
@@ -55,6 +55,7 @@
 				},
 				// Called by the main commitContent call on dialog confirmation.
 				commit: function (element) {
+					element.$.dataset["ckeSavedSrc"] = this.getValue();
```

Comment:
:+1:
could you please add the same fix in `static/djangocms_text_ckeditor/ckeditor/plugins/filerimage/dialogs/filerImageDialog.js`?

Reply:
Yes I do, I will do that, :)

Reply:
Done, :)